synapse/cortex.py

```diff
@@ -1014,8 +1014,15 @@ async def initServiceStorage(self):
         await self._bumpCellVers('cortex:storage', (
             (1, self._storUpdateMacros),
             (2, self._storLayrFeedDefaults),
+            (3, self._viewNomergeToProtected),
         ), nexs=False)
 
+    async def _viewNomergeToProtected(self):
+        for view in self.views.values():
+            nomerge = view.info.get('nomerge', None)
```

Comment:
Should this default to False and make it a required ( and auto populated ) field?

Reply:
I assume you mean the new `protected` option? I don't think it should be required but I can make it default to False in the schema and update this default value to be False.